speech/cloud-client/quickstart.py

```diff
@@ -35,14 +36,16 @@ def run_quickstart():
     # Loads the audio into memory
     with io.open(file_name, 'rb') as audio_file:
         content = audio_file.read()
-        sample = speech_client.sample(
-            content,
-            source_uri=None,
-            encoding='LINEAR16',
-            sample_rate_hertz=16000)
+        audio = types.RecognitionAudio(content=content)
+
+    config = types.RecognitionConfig(
+        encoding='LINEAR16',
```

Comment:
There is, but requires the additional import of `from google.cloud.speech import enums`.  I was keeping it consistent with the existing samples.